### PR TITLE
Add top/skip info to OData doc

### PR DESCRIPTION
**Pull request by johannahemminger**
https://mendix.atlassian.net/browse/TW-1942

### CODE REVIEW

File: content/en/docs/refguide/modeling/integration/published-odata-services/odata-query-options.md [hunk 1]
```diff
@@ -136,6 +136,8 @@ You can select which attributes and associations to return by specifying the `$s
 
 ## 7 Paging {#paging}
 
+Paging, or pagination, allows you to load data incrementally to better handle large amounts of data. See [Pagination](https://learn.microsoft.com/en-us/odata/client/pagination) in Microsoft's OData documentation for more information.
```
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/integration/published-odata-services/odata-query-options.md]: I think we should not use this link https://learn.microsoft.com/en-us/odata/client/pagination
First it's about microsoft's OData client, and there will be very few customers that use that client.
And second OData has moved from being a Microsoft standard to being an Oasis standard, so links to Microsoft in this context may suggest that we still treat it as a non-open standard.

File: content/en/docs/refguide/modeling/integration/published-odata-services/published-odata-resource.md [hunk 2]
```diff
@@ -191,6 +193,12 @@ The *Deletable* capability was introduced in Studio Pro [9.13.0](/releasenotes/s
 
 ## 10 Query Options {#query-options}
 
-Select the options that you would like to include in each OData capability.
+Select the options to include for the **Readable** OData capability.
 
 * **Countable** – required for getting the total number of records
+* **Top supported** – required when **Use paging** is selected.  This option supports the $top query, which requests the number of items in the queried collection to be included in the result.
```
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/integration/published-odata-services/published-odata-resource.md]: Can we put `required when **Use paging** is selected` at the end of this line? It's not as important as the rest of this line, so having that come first would improve readability.

Comment (joostverhoog) [inline content/en/docs/refguide/modeling/integration/published-odata-services/published-odata-resource.md]: "This option supports the $top query" -> "This option indicates whether clients can specify that they want to retrieve only a limited number of items."

Saying that the option supports something is not correct, my suggestion tries to rewrite it.

File: content/en/docs/refguide/modeling/integration/published-odata-services/published-odata-resource.md [hunk 3]
```diff
@@ -191,6 +193,12 @@ The *Deletable* capability was introduced in Studio Pro [9.13.0](/releasenotes/s
...
 
-Select the options that you would like to include in each OData capability.
+Select the options to include for the **Readable** OData capability.
 
 * **Countable** – required for getting the total number of records
+* **Top supported** – required when **Use paging** is selected.  This option supports the $top query, which requests the number of items in the queried collection to be included in the result. 
+* **Skip supported** – required when **Use paging** is selected. This option supports the $skip query, which requests the number of items in the queried collection that are to be skipped and not included in the result.
+
+For more information, see the following:
+* [Pagination](https://learn.microsoft.com/en-us/odata/client/pagination) in Microsoft's OData documentation
```
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/integration/published-odata-services/published-odata-resource.md]: I would like to have this link removed. The next one is fine.

File: content/en/docs/refguide/modeling/integration/published-odata-services/odata-query-options.md (hunk 1 above)
Comment (johannahemminger) [reply content/en/docs/refguide/modeling/integration/published-odata-services/odata-query-options.md]: Makes sense, thanks.

**Pull request merged**